src/main/java/seedu/address/logic/commands/AddPropertyToSellCommand.java

```diff
@@ -0,0 +1,72 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_HOUSING_TYPE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_POSTAL_CODE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_SELLING_PRICE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_UNIT_NUMBER;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Property;
+
+/**
+ * Adds a property to the list of properties to sell for a specific contact.
+ */
+public class AddPropertyToSellCommand extends Command {
+    public static final String COMMAND_WORD = "addSell";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a property to the list of properties to sell"
+            + "for the specific contact."
```

Comment:
Could I suggest "for this specific person"?

Rationale: We are going to call them Person, as per the Person class